.vscode/memory-inspector.code-snippets

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file is named `memory-inspector.code-snippets`. Any particular reason?

Reply:
No particular reason at all, I'm pretty sure this was an oversight, I'll rename the file. Thank you for catching that!

Reply:
I pushed an update that fixes the incorrect filename and also adapts the EPL copyright header to the correct MIT one.